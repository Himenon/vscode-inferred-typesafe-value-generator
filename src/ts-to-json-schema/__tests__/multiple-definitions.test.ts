import * as path from "path";
import * as Module from "../index";

const basePath = path.join(__dirname, "./sample-code");

describe.skip("Multi Type Test", () => {
  test("Load files directly", () => {
    const schema = Module.generateJsonSchema({
      files: [path.join(basePath, "./sample.ts")],
      typeName: {
        isUnknown: false,
        firstTypeName: "SameTypeConfig",
        isArray: false,
        intersectinoTypeNames: [],
        isIntersectionType: false,
      },
      basePath,
    });
    expect(schema).toMatchSnapshot();
  });
});
