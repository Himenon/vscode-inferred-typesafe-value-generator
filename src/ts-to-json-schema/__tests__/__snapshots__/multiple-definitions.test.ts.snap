// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Multi Type Test Load files directly 1`] = `
Object {
  "errorMessage": "Type \\"Config\\" has multiple definitions.",
  "jsonSchema": undefined,
}
`;
